spec/models/person_spec.rb

```diff
@@ -53,22 +53,22 @@
 
     describe '.find_person_from_guid_or_username' do
       it 'searchs for a person if id is passed' do
-        expect(Person.find_from_guid_or_username(:id => @person.guid).id).to eq(@person.id)
+        expect(Person.find_from_guid_or_username(id: @person.guid).id).to eq(@person.id)
       end
 
       it 'searchs a person from a user if username is passed' do
-        expect(Person.find_from_guid_or_username(:username => @user.username).id).to eq(@user.person.id)
+        expect(Person.find_from_guid_or_username(username: @user.username).id).to eq(@user.person.id)
       end
 
       it 'throws active record not found exceptions if no person is found via id' do
         expect{
-          Person.find_from_guid_or_username(:id => "2d13123")
+          Person.find_from_guid_or_username(id: "2d13123")
         }.to raise_error ActiveRecord::RecordNotFound
       end
 
       it 'throws active record not found exceptions if no person is found via username' do
         expect{
-          Person.find_from_guid_or_username(:username => 'michael_jackson')
+          Person.find_from_guid_or_username(username: 'michael_jackson')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.